### ISSUE TITLE
Add zome function that returns all the evolutions of a key

### ISSUE THREAD

**Issue by mjbrisebois** [holochain/deepkey#27]
A zome function that returns all the evolutions of a key when given any key from that lineage.

```rs
#[hdk_extern]
pub fn get_key_lineage(
    key_bytes: ByteArray<32>,
) -> ExternResult<Vec<KeyBytes>> {
    // ...
}
```

**Comment by abe-njama** [comment]
@jost-s @maackle does this cover everything that we need in Holochain to be able to [add the call to get lineage of agent keys](https://github.com/holochain/holochain/issues/4023)?

**Comment by maackle** [comment]
@abe-njama this issue and https://github.com/holochain/deepkey/issues/28, when completed, will cover our needs for DPKI, and both should be implemented by #29
